tasks/run-tower-job.yaml

```diff
@@ -109,7 +109,7 @@
       extra_vars: >-
         {{ tower_job_extra_vars
          | insert_unvault_string
-         | combine({"output_dir": "/tmp/output-" ~ vars.anarchy_subject.vars.job_vars.uuid})
+         | combine({"output_dir": __meta__.deployer.output_dir} if __meta__.deployer.output_dir is defined else { combine({"output_dir": "/tmp/output-" ~ vars.anarchy_subject.vars.job_vars.uuid}) }
```

Comment:
syntax seems off
i can't follow opening ( { and closing ) }

Isn't "if is defined "  equivalent to using `default()`?
For readability use a var:

```yaml
vars:
  _output_dir: >-
    {{ __meta__.deployer.output_dir
    | default("/tmp/output-" ~ vars.anarchy_subject.vars.job_vars.uuid )
    }}
awx.awx.tower_job_template
  extra_vars: >-
    {{ tower_job_extra_vars
     | insert_unvault_string
     | combine({"output_dir":  _output_dir})